### PR TITLE
[Enhancement] Move the zonal ppg and avset naming to naming module

### PR DESCRIPTION
**Pull request by KimForss**
## Problem
Currently the customer have limited control on zonal ppg and avset naming

## Solution
Create the zonal naming in the naming module

## Tests
<Please provide steps to test the PR>

## Notes
<Additional comments for the PR>

### CODE REVIEW

File: deploy/terraform/terraform-units/modules/sap_namegenerator/variables_local.tf [hunk 1]
```diff
@@ -166,7 +165,7 @@ variable region_mapping {
     "centralindia"       = "cein"
     "centralus"          = "ceus"
     "eastasia"           = "eaas"
-    "westeurope"             = "eaus"
+    "westeurope"         = "eaus"
```
Comment (hdamecharla) [inline deploy/terraform/terraform-units/modules/sap_namegenerator/variables_local.tf:168]: @kimforss, this would be "eastus"

**Review by hdamecharla — COMMENTED** [review]
Hi @KimForss, in ```deploy/terraform/terraform-units/modules/sap_namegenerator/variables_local.tf```, the region mapping on  line#168 should read "eastus" = "eaus" 🙏🏽

File: deploy/terraform/terraform-units/modules/sap_namegenerator/variables_local.tf (hunk 1 above)
Comment (KimForss) [reply deploy/terraform/terraform-units/modules/sap_namegenerator/variables_local.tf]: Edit Replace in Files mistake :)

**Review by hdamecharla — DISMISSED** [review]
Hi @KimForss, Approved #1094 👍🏾

**Pull request merged**